lib/cli/test/fixtures/meteor/.gitignore

```diff
@@ -1 +0,0 @@
-node_modules/
```

Comment:
Why all these changes on `lib/cli/test/fixtures/meteor/*`?

Reply:
I guess it's from a wrong merge/rebase 🤔

Reply:
No I deleted it, because it broke

Reply:
🆗  all theses fixtures will be 🗑️